src/main/java/com/example/dto/user/UserLoginResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package com.example.dto.user;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
```suggestion
@Getter
@Setter
```

check if you need these annotations